ajax/libs/agility/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "agility",
+  "description": "Javascript MVC for the \"write less, do more\"",
+  "homepage": "https://github.com/arturadib/agility",
+  "version": "0.1.3",
+  "licences": [
```

Comment:
Please use `licence` instead of `licences` since there is only MIT license here, thanks.